src/main/java/com/marklogic/mgmt/admin/AdminManager.java

```diff
@@ -81,7 +81,24 @@ public boolean execute() {
                 headers.setContentType(MediaType.APPLICATION_JSON);
                 HttpEntity<String> entity = new HttpEntity<>(payload, headers);
                 try {
-                    ResponseEntity<String> response = getRestTemplate().exchange(uri, HttpMethod.POST, entity, String.class);
+					boolean setNullUsername = false;
```

Comment:
Use a try/finally for this to ensure that the adminConfig is updated. And null out RestTemplate before/after the call in case adminConfig is updated. 

To avoid duplication, should be able to make a method that does this work and then returns an indication of whether username or password was modified (probably worth making a small inner class to hold two boolean values). And make a private method for the "finally" logic as well.